init.fish

```diff
@@ -123,3 +125,7 @@ abbr -a gfht       git flow hotfix track
 abbr -a gfst       git flow support track
 
 abbr -a gfp        git flow publish
+
+# Mark git plugin as initialized
+set -Ux __git_plugin_initialized (date)
+set -Ux __git_plugin_abbreviations (cat (status -f) | grep '^abbr\s\+-a\s\+\S\+' | awk '{print $3}')
```

Comment:
```suggestion
set -Ux __git_plugin_abbreviations (cat (status -f) | sed -n -e 's/^abbr\s\+-a\s\+\(\S\+\).*/\1/p')
```

However, it would likely be cleaner to create our own function that adds an abbreviation and appends the name to `__git_plugin_abbreviations` at the same time.  Then we don't have any external dependencies.

Reply:
@jhillyerd I like that idea :+1: